docs/docs/features/studio.md

```diff
@@ -0,0 +1,109 @@
+---
+title: ZMK Studio
+sidebar_label: Studio
+---
+
+:::warning[Alpha Feature]
+
+ZMK Studio support is in alpha. Although best efforts are being made, keeping compatibility during active development is not guaranteed.
+
+:::
+
+ZMK Studio provides runtime update functionality to ZMK powered devices. Studio is still under active development, and is not yet ready for casual end user use.
+
+## Building
+
+Building for ZMK Studio involves two main additional items.
+
+- Build with the `studio-rpc-usb-uart` snippet to enable the endpoint used for Studio communication over USB.
+- Enable the `ZMK_STUDIO` Kconfig setting.
+
+### GitHub Actions
+
+First add a `studio-rpc-usb-uart` to the `snippet` property of your build configuration. For a split keyboard, you should do this _only_ for your central/left side, e.g.:
+
+```
+---
+include:
+  - board: nice_nano_v2
+    shield: corne_left
+    snippet: studio-rpc-usb-uart
+  - board: nice_nano_v2
+    shield: corne_right
+```
+
+Next, enable the `ZMK_STUDIO` Kconfig symbol, for example by adding the following line to your .conf file:
+
+```
+CONFIG_ZMK_STUDIO=y
+```
+
+### Local Build
+
+When building locally, use the `-S` parameter to include the `studio-rpc-usb-uart` snippet. Instead of adding it to your config file, you can also append the `ZMK_STUDIO` Kconfig as an additional CMake argument, e.g.:
+
+```bash
+west build -d build/cl_studio -b nice_nano_v2 \
+  -S studio-rpc-usb-uart -- -DSHIELD=corne_left -DCONFIG_ZMK_STUDIO=y
+```
+
+## Behaviors
```

Comment:
```suggestion
## Including Unused Behaviors
```
I think it would be good to make the "unused" part explicit in the header for people who skim.

Reply:
Updated, but used the "Unreferenced" terminology instead.